support/images/envvars.go

```diff
@@ -0,0 +1,26 @@
+package images
+
+// Image environment variable constants
+const (
+	CAPIEnvVar                  = "IMAGE_CLUSTER_API"
+	AgentCAPIProviderEnvVar     = "IMAGE_AGENT_CAPI_PROVIDER"
+	AWSEncryptionProviderEnvVar = "IMAGE_AWS_ENCRYPTION_PROVIDER"
+	AWSCAPIProviderEnvVar       = "IMAGE_AWS_CAPI_PROVIDER"
+	AzureCAPIProviderEnvVar     = "IMAGE_AZURE_CAPI_PROVIDER"
+	KubevirtCAPIProviderEnvVar  = "IMAGE_KUBEVIRT_CAPI_PROVIDER"
+	KonnectivityEnvVar          = "IMAGE_KONNECTIVITY"
```

Comment:
can we add the `hypershift-operator` to the const list as well?

Reply:
it wouldn't be used anywhere because the hypershift operator deployment takes that directly as an image